test/Interop/Cxx/stdlib/std-unique-ptr.swift

```diff
@@ -0,0 +1,42 @@
+// RUN: %target-run-simple-swift(-I %S/Inputs -Xfrontend -enable-experimental-cxx-interop)
+//
+// REQUIRES: executable_test
+
+import StdlibUnittest
+import StdUniquePtr
+#if os(Linux)
+import CxxStdlib
+// FIXME: import CxxStdlib.string once libstdc++ is split into submodules.
```

Comment:
That's not going to happen. Can you just import CxxStdlib?

Reply:
We have a bunch of existing tests that do this, I'll update them to just import CxxStdlib.